### PR TITLE
Add share GIF

### PR DESCRIPTION
**Pull request by tiagomartinho**
Addresses bug:
https://phabricator.wikimedia.org/T139519

### CODE REVIEW

File: NYTPhotoViewer/NYTPhotosViewController.m [hunk 1]
```diff
@@ -275,7 +283,7 @@ - (void)actionButtonTapped:(id)sender {
     }
     
     if (!clientDidHandle && (self.currentlyDisplayedPhoto.image || self.currentlyDisplayedPhoto.imageData)) {
-        UIImage *image = self.currentlyDisplayedPhoto.image ? self.currentlyDisplayedPhoto.image : [UIImage imageWithData:self.currentlyDisplayedPhoto.imageData];
+        NSData *image = self.currentlyDisplayedPhoto.image ? UIImagePNGRepresentation(self.currentlyDisplayedPhoto.image) : self.currentlyDisplayedPhoto.imageData;
```
Comment (joewalsh) [inline NYTPhotoViewer/NYTPhotosViewController.m]: Could this be `id image =` and allow either the `UIImage` object or the `NSData` object to be passed to the activity VC? That way you don't have to use `UIImagePNGRepresentation()`

Comment (tiagomartinho) [reply NYTPhotoViewer/NYTPhotosViewController.m]: Yes @joewalsh ! Thanks for the information, I will add it right now.

**Comment by joewalsh** [comment]
Thanks!

**Comment by tiagomartinho** [comment]
Thank you!

**Pull request merged**